security/experimental_authenticators.rst

```diff
@@ -493,7 +493,7 @@ authenticator, you would initialize the passport like this::
             return new Passport($user, new PasswordCredentials($password), [
                 // $this->userRepository must implement PasswordUpgraderInterface
                 new PasswordUpgradeBadge($password, $this->userRepository),
-                new CsrfTokenBadge('login', $csrfToken);
```

Comment:
please change this to a comma instead, thanks!

Reply:
Done ✅